README.md

```diff
@@ -33,7 +33,7 @@ func main() {
 
 	engine := wazero.Engine()
 
-	module, err := engine.New(ctx, host, guest, &wapc.ModuleConfig{
+	module, err := engine.New(ctx, hostCall, code, &wapc.ModuleConfig{
```

Comment:
thanks, though I'd rename the other way as we have `host, guest` in the codebase

Reply:
Thanks, changed to renaming the others